src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/security/AuthorisedCaseSearchOperation.java

```diff
@@ -93,11 +93,11 @@ private CrossCaseTypeSearchRequest createAuthorisedSearchRequest(List<CaseTypeDe
         List<String> authorisedCaseTypeIds =
             authorisedCaseTypes.stream().map(CaseTypeDefinition::getId).collect(Collectors.toList());
 
-        return new CrossCaseTypeSearchRequest.Builder()
+        // clone CCT search request object :: then replace case type list
+        return new CrossCaseTypeSearchRequest.Builder(originalSearchRequest)
             .withCaseTypes(authorisedCaseTypeIds)
-            .withSearchRequest(originalSearchRequest.getElasticSearchRequest())
+            // NB: preserve original MultiCaseType processing instruction
             .withMultiCaseTypeSearch(originalSearchRequest.isMultiCaseTypeSearch())
```

Comment:
i guess builder is preserving original isMultiCaseTypeSearch() right?

Reply:
The new clone builder copies everything, but the flow up call to `.withCaseTypes(authorisedCaseTypeIds)` would reset the `isMultiCaseTypeSearch` flag if the filtered list of `authorisedCaseTypeIds` now only contained 1 value.  

However the previous code always preserved `isMultiCaseTypeSearch` flag (because of the order the statements are run):  therefore I have retained the additional call to set the  withMultiCaseTypeSearch after the call to withCaseTypes.

Reply:
Make sense. Thanks!